docs/spec/index.md

```diff
@@ -20,8 +20,7 @@ There are two types of files used in the TODS standard:
 | stops_supplement.txt | Supplement | Supplements and modifies GTFS [stops.txt](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#stopstxt) with internal stop locations, waypoints, and other non-public stop information.|
 | stop_times_supplement.txt | Supplement | Supplements and modifies GTFS [stop_times.txt](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#stop_timestxt) with non-public times at which trips stop at locations, `stop_times` entries for non-public trips, and related information. |
 | routes_supplement.txt | Supplement | Supplements and modifies GTFS [routes.txt](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#routestxt) with internal route identifiers and other non-public route identification. |
-| runs_pieces.txt | TODS-Specific | Defines daily personnel schedules within a feed. |
-| run_events.txt | TODS-Specific | Defines other scheduled activities to be performed by a member of personnel during a run. |
+| run_events.txt | TODS-Specific | Lists all trips and other scheduled activities to be performed by a member of personnel during a run. |
 
 _The use of the Supplement standard to modify other GTFS files is not yet formally adopted into the specification and remains subject to change. Other files may be formally adopted in the future._
```

Comment:
We should remove this line when we merge

Reply:
@safrazier17 Did you mean the `_The use of the Supplement standard to modify other GTFS files is not yet formally adopted into the specification and remains subject to change. Other files may be formally adopted in the future._` line?

I know there was some interest in removing it, but per #55 / #61 / the working group meetings, we opted to leave it in, since highlighting a forward-looking structure would promote generality / reuse of code in implementations as built by consumers.

Reply:
Oh I see. I thought it was a reference to the now-adopted files. But it is referring to other GTFS files, like calendar.txt. That makes sense, although I think it could maybe be a bit clearer. `TODS Supplements for other GTFS files may be formally adopted in the future, but only files explicitly listed here are part of the current version of TODS.` Maybe?

Reply:
That works for me, although I'm not sure how that fits into the broader scope of things given that (a) it wasn't officially proposed during the vote itself; and (b) that was technically already approved as part of #61's PR, not this one. If we don't want to pursue for the sake of governance, perhaps we add that tweak to 2.1's planned changes.